testbed/HIL_Firmware/common/include/dac.h

```diff
@@ -0,0 +1,32 @@
+#ifndef DAC_H
+#define DAC_H
+
+#include "driver/spi_master.h"
+
+#define BYTE_0 0x02
```

Comment:
You have so many of these masks. See if you can consolidate them into common or be more descriptive with what they are used for

Reply:
The 8 MSB on the external dac are constant so I made a macro, will rename